extensions/amp-brid-player/0.1/test/test-amp-brid-player.js

```diff
@@ -34,6 +34,18 @@ describes.realWin(
         bc.setAttribute('layout', 'responsive');
       }
 
+      // create config element if provided
+      if (config) {
+        const configElement = doc.createElement('script');
+        configElement.setAttribute('type', 'application/json');
+        if (typeof config == 'string') {
+          configElement.textContent = config;
```

Comment:
[eozmen410](https://github.com/eozmen410) do we need to make this assignment trusted types compatible?